.github/workflows/cd.yml

```diff
@@ -0,0 +1,118 @@
+          echo is_release=$(if [[ $CURRENT_OPERATOR_VERSION == *"rc"* ]];then echo "false";else echo "true";fi) >> "$GITHUB_OUTPUT"
+          echo is_release=true >> "$GITHUB_OUTPUT"
+          echo version=$CURRENT_OPERATOR_VERSION >> "$GITHUB_OUTPUT"
+
+  # TODO uncomment
+  # create-images:
+  #   runs-on: ubuntu-latest
+  #   if: ${{ fromJSON(needs.setup.outputs.is_release) }}
+  #   needs: setup
+  #   steps:
+  #     - name: Checkout code
+  #       uses: actions/checkout@v3
+
+  #     - name: Log in to Quay.io
+  #       uses: redhat-actions/podman-login@v1
+  #       with:
+  #         username: ${{ secrets.QUAY_USERNAME }}
+  #         password: ${{ secrets.QUAY_PASSWORD }}
+  #         registry: quay.io
+
+  #     - name: Build and push operator image
+  #       working-directory: ./pelorus-operator
+  #       run: |
+  #         make podman-build
+  #         make podman-push
+
+  #     - name: Build and push operator bundle image
+  #       working-directory: ./pelorus-operator
+  #       run: |
+  #         make bundle-build
+  #         make bundle-push
+
+  create-pr:
+    runs-on: ubuntu-latest
+    if: ${{ fromJSON(needs.setup.outputs.is_release) }}
+    needs: setup
+    # TODO uncomment
+    # needs: [setup, create-images]
+    steps:
+      # TODO uncomment
+      # - name: Checkout marketplace code
+      #   uses: actions/checkout@v3
+      #   with:
+      #     repository: ${{ env.FORK }}/community-operators-prod
+      #     token: ${{ secrets.FORK_TOKEN }}
+
+      # - name: Update fork repo
+      #   run: |
+      #     git remote add upstream https://github.com/redhat-openshift-ecosystem/community-operators-prod.git
+      #     git fetch upstream
+      #     git rebase upstream/main
+      #     git push origin main
+
+      # - name: Create branch
+      #   run: |
+      #     git checkout -b pelorus-operator-${{ needs.setup.outputs.version }}
+      #     mkdir -p operators/pelorus-operator/${{ needs.setup.outputs.version }}
+
+      # - name: Download operator code
+      #   uses: actions/download-artifact@v3
+      #   with:
+      #     name: operator-code
+      #     path: operators/pelorus-operator/${{ needs.setup.outputs.version }}
+      #     # TODO remove
+      #     # cp -r ../pelorus/pelorus-operator/bundle/* operators/pelorus-operator/${{ needs.setup.outputs.version }}
+
+      # - name: Push branch
+      #   run: |
+      #     git config user.name github-actions # TODO create bot account
+      #     git config user.email github-actions@github.com # TODO create bot account
+      #     git add -A
+      #     git commit --signoff -m "operator pelorus-operator (${{ needs.setup.outputs.version }})"
+      #     git push origin pelorus-operator-${{ needs.setup.outputs.version }}
+
+      - name: Create PR
+        # TODO change to upstream repo and master branch
+        uses: "mateusoliveira43/pelorus/.github/workflow_templates/create-market-place-pr@fix/release-steps-and-automation"
```

Comment:
because I do not want to clone the repo here just to use this file (I do clone in the openshifty repo on this job, not ours)

it was my user just to test out, it will run against our ORG and master